doc/architecture.rst

```diff
@@ -318,9 +318,10 @@ and uses it to sign requests to OSDs and to metadata servers in the cluster.
 
 
 The ``cephx`` protocol authenticates ongoing communications between the client
-machine and the Ceph servers. Each message sent between a client and server,
-subsequent to the initial authentication, is signed using a ticket that the
-monitors, OSDs and metadata servers can verify with their shared secret.
+machine and the Ceph servers. After initial authentication, each message sent
```

Comment:
s/machine//
s/and the/and/

the second is marginal

Reply:
Accepted.

Reply:
Well, I'm leaving the "the" in the second part for the sake of completely parallel construction.

Reply:
:p

Reply:
Subsequent developments related to the incorporation of the word "daemon" where previously "server" was have obviated the need for this definite article. To the degree that there was any agon in this matter, you can consider this a victory.

Reply:
I HAVE TEH_POWER